resources/migrations/20210201115807-add-form-data.up.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE form_template ADD formdata jsonb;
```

Comment:
formdata sounds pretty generic: could fields move inside formdata? if not, could we rename formdata to something like metadata?

Reply:
I would move stuff inside formdata but I don't want to do it in this PR.

Reply:
I think the gold style for PG + JSON model is to only have the id + potentially some indexed columns + at most one blob.

Reply:
agreed! follow-up PR sounds good.

Reply:
I put it here because it's not urgent https://github.com/CSCfi/rems/issues/2561